packages/blade/src/utils/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -21,7 +21,7 @@ import { Button } from '~components/Button';
 const getBladeVersion = (): string => {
   // We don't publish codesandbox ci on master so version is not present
   const isMaster = process.env.GITHUB_REF === 'refs/heads/master';
-  const sha = process.env.GITHUB_SHA ?? '6f76c2e1d36a5f0076ee76470b36cd54b7d4801b'; // @todo: for development, remove before merging
```

Comment:
And I, in fact, did not remove it before merging 😆

Reply:
Was it causing any issues?